apple/REAModule.h

```diff
@@ -1,14 +1,24 @@
+#ifdef RCT_NEW_ARCH_ENABLED
+#import <rnreanimated/rnreanimated.h>
+#else
 #import <React/RCTBridgeModule.h>
+#endif
```

Comment:
Can we add a newline here for better readability?
```suggestion
#endif // RCT_NEW_ARCH_ENABLED

```

Reply:
Ok I'll add it everywhere